silver/tests/spec/test_payment_method.py

```diff
@@ -1,49 +1,330 @@
+import sys
+from copy import deepcopy
+
+from rest_framework import permissions, status
 from rest_framework.reverse import reverse
+from six import iteritems
 
 from silver.api.serializers import PaymentMethodSerializer
 from silver.models import PaymentMethod
+from silver.api.views import PaymentMethodList, PaymentMethodDetail
 from silver.tests.factories import CustomerFactory, PaymentMethodFactory
+from silver.tests.spec.test_transactions import register
 from silver.tests.spec.util.api_get_assert import APIGetAssert
 
 
 class TestPaymentMethodEndpoints(APIGetAssert):
     serializer_class = PaymentMethodSerializer
 
-    def test_filter_processor(self):
-        customer = CustomerFactory.create()
-        payment_method = PaymentMethodFactory.create(
-            customer=customer
-        )
+    def setUp(self):
+        self.customer = CustomerFactory.create()
+
+        super(TestPaymentMethodEndpoints, self).setUp()
+
+    def create_payment_method(self, *args, **kwargs):
+        method = PaymentMethodFactory.create(*args, **kwargs)
 
         # mysql does not store fractional time units but the object
         # created will have them so we can't use it directly
         #  to check the output
-        payment_method.refresh_from_db()
+        method.refresh_from_db()
```

Comment:
The comment above. When creating and serializing vs api serialization are 2 different things.
If I serialize the object directly it will have a datetime with microseconds if I get it from the db it won't have that data and won't be identical.